src/lib/mixer/mixer.h

```diff
@@ -707,7 +707,18 @@ class MultirotorMixer : public Mixer
 	 *
 	 * @param[in]  val   The value
 	 */
-	void			set_thrust_factor(float val) override { _thrust_factor = val; }
+	void			set_thrust_factor(float val) override
+	{
+		if (val < 0) {
```

Comment:
math::constrain

Reply:
@dagar Thanks. I somehow had in mind that it was not available on IO but then that's not the case.